community/contributing/pr_workflow.rst

```diff
@@ -204,8 +204,13 @@ Therefore, the command that you should (almost) always use is:
 
     $ git pull --rebase upstream master
 
+You may configure git to use ``--rebase`` option automatically for you when you pull, with:
```

Comment:
This should be a `.. tip::`.

I'm not sure it's good advice though, it's important that people are aware of when they are rebasing or not. There are situations where `pull --rebase` is NOT the right thing to do.

Reply:
What are those situations ?
docs doesn't specify them and advocate for rebase option usage without clear distinction.
should we document them ?